WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -519,8 +525,11 @@ public void onPostButtonClicked(int buttonType, PostModel post) {
                 break;
             case PostListButton.BUTTON_TRASH:
             case PostListButton.BUTTON_DELETE:
+                // TODO: Async: Update this behavior to handle pressing Delete/Trash while the post has uploading media
+                // Currently, the button is tappable while media are uploading, but nothing happens
+
                 // prevent deleting post while it's being uploaded
-                if (!PostUploadService.isPostUploading(post)) {
+                if (!PostUploadService.isPostUploadingOrQueued(post)) {
```

Comment:
Since this needs design input and we're putting that off for now, I'd rather leave it obviously not working than add what looks like a finished solution - and rely on that plus the TODO and our own list of tasks to track updating this once we have a final design to implement for the post list. Does that make sense?